### PR TITLE
epstool: add test and missing runtime dependency

### PR DESCRIPTION
**Pull request by xu-cheng**
also add test.eps fixture

### CODE REVIEW

File: Library/Homebrew/test/fixtures/test.eps [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+20 selectfont
+72 500 moveto
+(Hello World!) show
+showpage
+%%Trailer
+cleartomark
+countdictstack
+exch sub { end } repeat
+restore
+%%EOF
```
Comment (mikemcquaid) [inline Library/Homebrew/test/fixtures/test.eps]: Any chance se can shrink this? No worries if not, it's not that big.

Comment (xu-cheng) [inline Library/Homebrew/test/fixtures/test.eps]: I'm not too familiar to eps standard. This one is generated by `ps2eps` where ps file is from wikipedia.

**Comment by xu-cheng** [comment]
Failed at the same reason as https://github.com/Homebrew/homebrew/pull/36902. Already reported this bug to Github by email. And I need go to sleep.

**Comment by mikemcquaid** [comment]
> Failed at the same reason as #36902. Already reported this bug to Github by email. And I need go to sleep.

Feel free to report GitHub bugs to me directly and I can deal with them internally.

**Comment by xu-cheng** [comment]
@mikemcquaid Instead of fixing the bug, Github now reported this:

```
$ curl https://github.com/Homebrew/homebrew/pull/36905.patch
Content containing PDF or PS header bytes cannot be rendered from this domain for security reasons.
```

**Comment by mikemcquaid** [comment]
@xu-cheng Can we encode it differently or something I wonder rather than putting the magic bytes as plain text?

**Comment by xu-cheng** [comment]
AFAIK, magic bytes are mandatory(although, I'm not 100% sure). Is there anyone familiar with postscript/pdf specs?

**Comment by mikemcquaid** [comment]
Oh, sure, I just mean something like ROT13 them in the string and then de-ROT13 them in Ruby?

**Comment by xu-cheng** [comment]
I just try to make Github treat them as binary file. But no luck. Github doesn't seem to respect `.gitattributes` file.

**Comment by jacknagel** [comment]
If the only problem is that github won't render it as a patch, can't we just ignore that and commit it? I assume this file won't change often.

**Comment by mikemcquaid** [comment]
Yeh, let's do that and just put the PS file into a separate fixture too.

**Comment by xu-cheng** [comment]
OK, creating a new PR with ps and eps fixtures.

**Pull request closed without merging**